### ISSUE TITLE
Heterogeneous condensation

### ISSUE THREAD

**Issue by greffinea** [philipmorrisintl/aerosolved#7]
Hi，I'm a freshman,I want to use aerosolved to simulate heterogeneous condensation in the turbine.Someone could give me some advice about how to control the particles at the beginning.(I'm very confused about where and what I should change.)Thanks a lot.

**Comment by bluelee333633** [comment]
it seems you need to set the particle properties at the beginning

发自我的 iPhone

在 2022年11月10日，10:28，greffinea ***@***.***> 写道：

﻿

Hi，I'm a freshman,I want to use aerosolved to simulate heterogeneous condensation in the turbine.Someone could give me some advice about how to control the particles at the beginning.(I'm very confused about where and what I should change.)Thanks a lot.

**Comment by greffinea** [comment]
yes,I want to take the Constant particles at the beginning,but I don't know where,can you give me some advice.Thanks.
